src/main/webapp/app/shared/services/auth.service.ts

```diff
@@ -10,20 +10,36 @@ export async function getAuthToken(firebaseToken: string): Promise<void> {
   const headers = {
     [FIREBASE_AUTH_HEADER_NAME]: firebaseToken
   };
-  const response = await axios.post('/api/firebase/authenticate', null, {
+  const response = await axios.post('/api/authenticate/firebase', null, {
     headers
   });
-  const jwtToken: string = response.data['id_token'];
+  const jwtToken: string = response.data['accessToken'];
+  const refreshToken: string = response.data['refreshToken'];
   Storage.local.set(AUTH_TOKEN_KEY, jwtToken);
```

Comment:
change all `jwtToken` naming to `accessToken` to avoid confussion

Reply:
resolve in 925de3b100431105e346fe573c227ebf409e3927